articles/creating-a-location-tracking-app-using-firebase-and-google-maps-in-android/index.md

```diff
@@ -0,0 +1,386 @@
+
+## Introduction
+
+In this tutorial, you will learn the steps in creating an application that uses Google Maps to give the precise location of another user's device. You shall primarily use Google Maps API to show the exact location. This will be achieved by writing the user's location on a Realtime Database using Firebase and read the location from another device.
+
+Location services have had many applications, especially social apps such as fitness apps, Pokemon Go. And hey, you can also find the location of your stolen device.
+
+By the end of this tutorial, you shall learn how to create two simple apps to aid with location tracking, how to work with Google Maps API and Firebase's Realtime Database.
+
+Comments will be added to the code snippets for better understanding.
+
+### Pre-requisites
+- A good understanding of Kotlin in Android development
+
+#### Reccomendation
+For this tutorial, we shall use real Android device(s) to test the application(s). There might arise a reason why you might test them using an emulator. This is achieved by mocking location data in the emulator. You can read more [here](https://medium.com/@msaudi/android-test-location-services-and-gps-with-fake-gps-data-mock-locations-in-emulators-or-real-df211de4d891)
+
+### Getting started
+##### Step 1: Create a new project
+Open Android Studio and create a new project. On the Select, a Project template, select the Google Maps Activity template
```

Comment:
On the Select, a Project template, `select` the Google Maps Activity template instead of the highlighted select add `choose`